### PR TITLE
Fix/change button color

### PR DESCRIPTION
**Pull request by ekaTolk1006**
Changed button color in the login modal.

### CODE REVIEW

**Review by voloshynajelena — APPROVED** [review]

File: src/styles/colors.less [hunk 1]
```diff
@@ -8,6 +8,7 @@
 @color-blue-sky: #a6c4c8;
 @color-forest: #007934;
 @color-lime: #c9dd03;
+@color-hex:#07676c;
```
Comment (voloshynajelena) [inline src/styles/colors.less:11]: where are you using this variable?

File: src/app/components/patient-tests-table/patient-tests-table.component.less [hunk 2]
```diff
@@ -223,7 +222,7 @@
   margin-right: 15px;
 }
 .count-checked-tests span {
-  color: #3f51b5;
+  color:#07676c;
```
Comment (voloshynajelena) [inline src/app/components/patient-tests-table/patient-tests-table.component.less:225]: Why you don't use variable from colors.less here?

File: src/app/components/header/help-menu/help-menu.component.less [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.mat-icon{
```
Comment (voloshynajelena) [inline src/app/components/header/help-menu/help-menu.component.less:1]: this style you need to place in app.component.less file

File: src/app/app.component.less [hunk 4]
```diff
@@ -3,10 +3,19 @@
...
   background-color: #07676c !important;
 }
-
-
+.mat-button.mat-primary,.button-icon{
+  color:  #07676c;
+}
 // end of theming
-
+.mat-mini-fab{
```
Comment (voloshynajelena) [inline src/app/app.component.less:10]: move this styles to the scope of theming (it is defined by comments above)

**Review by voloshynajelena — CHANGES_REQUESTED** [review]

File: src/app/components/header/help-menu/help-menu.component.less (hunk 3 above)
Comment (ekaTolk1006) [reply src/app/components/header/help-menu/help-menu.component.less:1]: If I add this style in the app. component, everywhere changing color. And I add this style only in this component, that style has only this button.
image
image

File: src/app/app.component.less (hunk 4 above)
Comment (ekaTolk1006) [reply src/app/app.component.less:10]: Done

File: src/app/components/patient-tests-table/patient-tests-table.component.less (hunk 2 above)
Comment (ekaTolk1006) [reply src/app/components/patient-tests-table/patient-tests-table.component.less:225]: Done

**Review by voloshynajelena — APPROVED** [review]

**Review by voloshynajelena — APPROVED** [review]

**Review by voloshynajelena — APPROVED** [review]

**Pull request merged**